hack/test/e2e.sh

```diff
@@ -325,7 +325,7 @@ function validate_rlimit_nofile {
 
 function validate_booted_secureboot {
   ${TALOSCTL} dmesg | grep "Secure boot enabled"
-  ${TALOSCTL} get securitystate -o json 
+  ${TALOSCTL} get securitystate -o json
```

Comment:
we can remove this now :sweat_smile:

Reply:
I overall feel we should move this to the integration tests... I think we should cut our shell scripts

Reply:
ahh yes, and fix how we detect secureboot in integration test. I'll do in a separate PR